src/bp/ui-studio/src/web/reducers/index.ts

```diff
@@ -1,6 +1,8 @@
 import { ModuleDefinition } from 'botpress/sdk'
 import { combineReducers } from 'redux'
 
+import { Bot } from '../../../../../../out/bp/common/typings'
```

Comment:
That typing is old and not used anymore I think, take the BotConfig from the sdk instead